docs/rust-client/public_account_interaction_tutorial.md

```diff
@@ -0,0 +1,652 @@
+# Interacting with Public Smart Contracts
+
+*Using the Miden client in Rust to interact with public smart contracts on Miden*
+
+## Overview
+
+In the previous tutorial, we built a simple counter contract and deployed it to the Miden testnet. However, we only covered how the deployer of a smart contract could interact with the smart contract they deployed. In this tutorial, we will cover how anyone can interact with a public smart contract on Miden.
```

Comment:
Can you add a paragraph here and explain what you are doing process-wise. 

We need to "download" the account from the chain in order to execute a local transaction against it. In the near future Miden will support network transactions, then the process will be much more like on traditional blockchains.